src/components/VideoPlayer/VideoPlayer.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+import { VideoPlayerAspectRatios, VideoPlayerTypes } from "./VideoPlayerTypes";
+
+export interface VideoPlayerProps {
+  /** Optional aspect ratio prop to control the sizing of the video player; if omitted, the video player defaults to `sixteen-by-nine` */
+  aspectRatio?: VideoPlayerAspectRatios;
+  /** Optional additional attributes passed to the video player iframe */
+  attributes?: { [key: string]: any };
+  /** Optional className you can add in addition to `video-player` */
+  className?: string;
+  /** Optional title to be added to the `<iframe>` element for improved accessibility; this title should describe in a few words the content of the video; if omitted, a generic title will be added */
+  iframeTitle?: string;
+  /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
+  modifiers?: string[];
+  /** Required YouTube or Vimeo video ID. This value can be pulled from a video's YouTube or Vimeo URL. */
+  videoId: string;
+  /** Required. Used to specify which video service is being used. */
+  videoType: VideoPlayerTypes;
+}
+
+export default function VideoPlayer(
+  props: React.PropsWithChildren<VideoPlayerProps>
+) {
+  const {
+    aspectRatio,
+    attributes = {},
+    className,
+    iframeTitle,
+    modifiers = [],
+    videoId,
+    videoType,
+  } = props;
+
+  let embedCode;
+  let iframeTitleFinal;
```

Comment:
Unsolicited Typescript advice - it's not a mistake, per se, but when possible, it's good practice to structure code to allow yourself to use `const` instead of `let`.

For example,  here you could avoid assigning `embedCode` here and just go `const embedCode = ` on L54, and change L29 to `iframeTitle = videoType === VideoPlayerTypes.Vimeo ? "Vimeo video player"  : "YouTube video player"` (and do a similar thing to videoSrc).

Reply:
@hokei Thanks for the suggestions. I made some changes.